scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -192,7 +192,10 @@ Param (
     [string] $AmqpSettingsEnabled = "true",
 
     [ValidateSet("true", "false")]
-    [string] $MqttSettingsEnabled = "true"
+    [string] $MqttSettingsEnabled = "true",
+
+    [ValidateNotNullOrEmpty()]
```

Comment:
don't need [ValidateNotNullOrEmpty] for switch type.

Reply:
Removed.